source/blog/2020-09-21-this-month-in-solidus-summer-2020.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: "This Month in Solidus: Summer 2020"
+date: 2020-09-21
+tags: Solidus
+author: Alessandro Desantis & Sean Denny
+cover_image: /blog/shared/images/solidus-monthly-update.png
+description: >
+  Dive in and see everything that happened in the Solidus world in summer 2020: new Klaviyo & PayPal
+  integrations, great progress for multichannel support and the usual core improvements!
+---
+
+Hello everyone, and welcome to the third edition of This Month in Solidus, August 2020 edition!
+
+You may have noticed we've paused this series for a couple of months: with the summer, the
+[conference](https://solidus.io/blog/2020/07/17/announcing-solidusconf-2020.html) and a lot of
+people being on vacation, the work has been progressing steadily but a bit more slowly, and we
+thought it would best to wait until we had more existing news to share! 
+
+### Klaviyo integration
+
+[The long-awaited Klaviyo integration is finally here!](https://github.com/solidusio-contrib/solidus_klaviyo) This extension brings the powerful marketing & analytics tools that Klaviyo provides to the Solidus ecosystem. 
+
+### PayPal Commerce Platform
+
+Our newest payment method for Solidus, [PayPal Commerce Platform](https://github.com/solidusio-contrib/solidus_paypal_commerce_platform), is here! This is the new default payment method for Solidus, and adds some handy features like instant checkout and easier onboarding.
+
+### Core updates this summer
+
+Here are some of the new features and enhancements introduced this summer: 
+
+- [The Order state machine was made configurable and simplified.](https://github.com/solidusio/solidus/pull/3542) This removes a lot of possible transitions - and complexity - from the state machine, and allows it to be configurable.
+- [Creating an order from the admin user page will now prefill the users address on the order.](https://github.com/solidusio/solidus/pull/3558) This should save some time when creating new orders.
+- [The ability to select multiple rows on admin tables was added.](https://github.com/solidusio/solidus/pull/3565) This is controlled by CSS classes, making it easy to drop this functionality in wherever you want.
+- [We've added an API endpoint for customer returns.](https://github.com/solidusio/solidus/pull/3579) Previously this didn't exist - now it does! 🎉
+- [An admin UI for adding stock locations to shipping methods was added.](https://github.com/solidusio/solidus/pull/3624) This allows you to associate shipping methods to stock locations, ensuring that a shipping method will only be available on whatever stock locations you designate.
+- [Validations will not run on order updates.](https://github.com/solidusio/solidus/pull/3645) You can disable this functionality with a config option, but it's desirable to ensure that everything is still valid for an order when it updates.
+- [Variant properties can now be set to apply to all variants.](https://github.com/solidusio/solidus/pull/3653)
```

Comment:
I think I understood this PR correctly, but my brain is pretty caffeine-deprived at the moment, so please double check me!